tasks/main.yml

```
@@ -85,6 +85,9 @@
     force_state_change: "{{ network_force_state_change | default(omit) }}"
     connections: "{{ network_connections | default([]) }}"
     __debug_flags: "{{ __network_debug_flags | default(omit) }}"
+    __ansible_managed: "{{ __network_new_header }}"
+  vars:
+    __network_new_header: "{{ lookup('template', 'get_ansible_managed.j2') }}"
```

This was my proposal (I also removed the `network` prefix here since it is scoped by being a parameter for the `network_connections` module):

```suggestion
    __header: "{{ __ansible_managed }}"
  vars:
    __ansible_managed: "{{ lookup('template', 'get_ansible_managed.j2') }}"
```

---

Thanks for the clarification, @tyll. So, you prefer to use `__header` for the network_connection parameter.

@richm, what do you think about sharing the parameter name among the roles?
For instance, I was planning to replace `ansible_managed_new` with `__ansible_managed` in the certificate role some time later (introduced by https://github.com/linux-system-roles/certificate/pull/112). Do we want to use `__header` for all the applicable parameters or we do not care using the different names?

---

> Thanks for the clarification, @tyll. So, you prefer to use `__header` for the network_connection parameter.

Yes.

> @richm, what do you think about sharing the parameter name among the roles? For instance, I was planning to replace `ansible_managed_new` with `__ansible_managed` in the certificate role some time later (introduced by [linux-system-roles/certificate#112](https://github.com/linux-system-roles/certificate/pull/112)). Do we want to use `__header` for all the applicable parameters or we do not care using the different names?

I realized that instead of `__ansible_managed` it should pobably be `__lsr_ansible_managed`, since it is an linux system roles internal variable to avoid potential conflicts with something else using `__ansible_managed`.